File: redis/golang/models/responses/get_response.go
package modelresponses

import modelentities "note-golang-redis/models/entities"

type GetResponse struct {
	Id   string `json:"id"`
	Test string `json:"test"`
}

func SetGetResponse(test1 modelentities.Test1) GetResponse {
	return GetResponse{
		Id:   test1.Id,
		Test: test1.Test,
	}
}

File: redis/rust/src/models/responses/get_response.rs
use serde::Serialize;
use crate::models::entities::test1::Test1;

#[derive(Debug, Serialize)]
pub struct GetResponse {
    pub id: String,
    pub test: String,
}

impl GetResponse {
    pub fn set_get_response(test1: Test1) -> Self {
        Self {
            id: test1.id,
            test: test1.test,
        }
    }
}